docs/vision.md

```diff
@@ -1,4 +1,28 @@
+
+
+## Hybrid computing beyond circuits
+Many quantum algorithms are hybrid, requiring both classical and quantum resources to work together in a hybrid computation architecture. This could be anything from syndrome extraction and measurement-based computing to variational parameter updates in VQE methods and orbital fragmentation methods in molecular simulation. Through the use of the Kirin compiler infrastructure, Bloqade embraces this philosophy of heterogeneous compute. Kirin programs are written as (compositions of) [kernels](https://en.wikipedia.org/wiki/Compute_kernel)-- subroutines that are intended to run on particular hardware (such as QPUs), or orchestrated to run on heterogeneous compute (such as a real-time classical runtime plus a QPU). These subroutines-- plus the built-in hybrid representations-- enable many key primitives, such as error correction.
+
+Additionally, the ability to compose functions together and to use typical classical programming structures like `if` and `while` enables many simplifications in writing raw circuit executions. In fact, `while` loops and the ability to dynamically allocate new memory (which is not known until runtime) enables many powerful subroutines and is natively enabled with bloqade's kernel-based representation; for example, see [this implementation](digital/examples/repeat_until_success.py) of a repeat-until-success program.
+
+## Analog, digital, logical: towards real quantum utility
+The first step in Bloqade was building out the analog mode SDK, designed to interface with QuEra’s cloud-accessible analog-mode neutral-atom quantum computer Aquila, as well as enable analysis and scientific discovery in analog quantum computing. But the journey should not stop there: real quantum utility is error corrected and requires robust algorithmic exploration and design of quantum primitives, in-depth analysis of near-term hardware performance and benchmarking, and building pipelines and hybrid architectures that are intended not just for today’s demonstrators but also for tomorrow’s utility-scale hardware. By introducing the next generation of Bloqade, we hope to enable this exploration by adding in support for near-term digital and intermediate-term logical representations of hybrid quantum computations.
+
+## Join us!
+
+If you are interested in contributing, please see the contribution page [here](quick_start/contrib.md). If you are interested in exploring more about neutral atom quantum computing, check out some analog tutorials [here](), and some circuit tutorials [here](). If you wish to work closer with QuEra, please feel free to reach out!
```

Comment:
reach out as in file an issue/PR/check out our careers page? Worth linking that here or making more explicit